src/pages/MovieListPage/MovieListPage.tsx

```diff
@@ -64,34 +58,23 @@ export default function MovieListPage() {
         navigate(`movies/${movieDetails.id}${window.location.search}`);
     }
 
-    function handleToggleDialog() {
-        setShowDialog(!showDialog);
+    function handleAddMovieButtonClick() {
+        navigate(`/new${window.location.search}`);
     }
 
-    function handleShowDialog(dialogOption: string, movieDetails?: MovieDetailsData) {
-        setCurrentDialog(dialogOption);
-
-        if (movieDetails) {
-            setSelectedMovie(movieDetails);
-        }
-
-        handleToggleDialog();
+    function handleEditMovieButtonClick(movieDetails: MovieDetailsData) {
+        navigate(`movies/${movieDetails.id}/edit${window.location.search}`);
     }
 
-    function handleSubmit(MovieData: MovieDetailsData) {
-        console.log(MovieData);
+    function handleDeleteMovieButtonClick(movieDetails: MovieDetailsData) {
+        navigate(`movies/${movieDetails.id}/delete${window.location.search}`);
     }
```

Comment:
We can have one function, that accept second parametr like enum EDIT or DELETE 
![image](https://github.com/user-attachments/assets/8a53f0bb-eea5-41ac-97d2-9d0aae6c110f)


because here we have only one difference